### ISSUE TITLE
"Voltage Control" issue in spkm v13.4

### ISSUE THREAD

**Issue by bhavtosh** [SmartPack/SmartPack-Kernel-Manager#81]

**Comment by sunilpaulmathew** [comment]
Hi,
Hanging phone should be an issue with Kernel, & no idea why Min/Max values don't reflect changes. It is strange.

**Comment by bhavtosh** [comment]
Ok but is it necessary for these 2 files to be there in kernel for spkm to allow voltage control to work OR UV_mV_table file is enough?
How did u test this feature?

Min/Max values change in voltage screen but when user goes back to main screen and comes back to voltage screen then default values r shown...It means changes are not saved by spkm?

**Comment by sunilpaulmathew** [comment]
> user goes back to m

Not at all. The two paths represent different devices (or at least kernels). 
I can't test any of the voltage related stuff here at all (no device).
& The values going back to default means, the new values applied by the app is not accepted by the kernel. There will be many reasons. e.g. the respective sysfs path is not writable, or the app is trying to write a wrong file and so on.
